modules/ob-defining-strategy-parameters.adoc

```diff
@@ -0,0 +1,45 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-build-strategies.adoc
+
+:_content-type: CONCEPT
+[id="ob-defining-strategy-parameters_{context}"]
+= Defining strategy parameters
+
+You can define strategy parameters in a `BuildStrategy` or `ClusterBuildStrategy` CR and set or modify the values of those parameters in your `Build` or `BuildRun` CR. You can configure or modify strategy parameters at build time when creating your build strategy.
+
+Before defining parameters for your strategy, understand the following points:
+
+* Define a list of parameters in the `spec.parameters` field of your build strategy CR. Each list contains a name, a description, a type, and optionally a default value, or defaults values for an array type. If no default value is set, then you must define a value in the `Build` or `BuildRun` CR.
+* Specify a string parameter in the strategy steps by using this syntax for `type=string: $(params.your-parameter-name)`. You can use string parameters in the `spec.steps` field of your build strategy. You can set a value for the `your-parameter-name` parameter in your `Build` or `BuildRun` CR that references your strategy. You can define the following string parameters based on your needs:
+** `image`: Use this parameter to define a custom tag, such as `golang:$(params.go-version)`
```

Comment:
I am also in favor of your suggestion. The only reason i did not prefer the table here because the parameters are very less and the description is also a one-liner.